2dcontext/tools/current-work-canvas.xhtml

```diff
@@ -1,50 +1,74 @@
-<?xml version="1.0" encoding="UTF-8"?><html xml:lang="en-US-x-hixie" xmlns="http://www.w3.org/1999/xhtml"><head><title>HTML Standard</title><link href="http://www.whatwg.org/style/specification" rel="stylesheet"/><link href="http://www.whatwg.org/images/icon" rel="icon"/><style>
```

Comment:
I presume we can't just use a git submodule here rather than checking in a large part of the HTML spec?

Reply:
I dunno, I thought we generally try to avoid submodules? Also spec.yaml has text that's supposed to sync up with the spec, it seems clearer for reviewers if it's checked in.

Reply:
We have plenty of other submodules used for such things; I thought it was required submodules that people didn't like. But yeah, I don't really care. *shrugs*